docker-compose.yml

```diff
@@ -1,23 +1,32 @@
 version: "3.9"
    
 services:
-  db:
-    image: postgres
+  postgres:
+    image: postgres:13.2-alpine
     volumes:
-      - ./data/db:/var/lib/postgresql/data
+      - ./data/postgres:/var/lib/postgresql/data
     env_file:
       - ./postgres/.env
+  mysql:
+    image: mysql
+    ports:
+      - "3306:3306"
+    volumes:
+      - ./data/mysql:/var/lib/mysql
+    env_file:
+      - ./mysql/.env
+    command: --default-authentication-plugin=mysql_native_password
```

Comment:
@abhijeetk783 this is what's needed to avoid manually altering the database with:

 ```ALTER USER 'root' IDENTIFIED WITH mysql_native_password BY 'password';`` 
 
 The above is not needed now since the mysql container will automatically create the users (see https://hub.docker.com/_/mysql and search for 'native')

Reply:
Understood, Thank you.